client/cl_main.lua

```diff
@@ -0,0 +1,322 @@
+RegisterNetEvent('qbx_management:client:EmployeeList', function(type)
+    local EmployeesMenu = {}
+    local accountName = QBX.PlayerData[type].name
+    local employees = lib.callback.await('qbx_management:server:getemployees', false, accountName, type)
+    for _, v in pairs(employees) do
+        EmployeesMenu[#EmployeesMenu + 1] = {
+            title = v.name,
+            description = v.grade.name,
+            event = 'qbx_management:client:ManageEmployee',
+            args = {
+                type = type,
+                player = v,
+                work = QBX.PlayerData[type]
+            }
+        }
+    end
+
+    EmployeesMenu[#EmployeesMenu + 1] = {
+        title = 'Return',
+        icon = 'fa-solid fa-angle-left',
+        event = 'qbx_management:client:OpenMenu',
+        args = type
+    }
+
+    lib.registerContext({
+        id = 'qbx_management_open_Manage',
+        title = type == 'gang' and 'Manage Gang Members - ' .. string.upper(QBX.PlayerData.gang.label) or 'Manage Employees - ' .. string.upper(QBX.PlayerData.job.label),
+        options = EmployeesMenu
+    })
+
+    lib.showContext('qbx_management_open_Manage')
+end)
+
+---@param data {source: number, player: table, work: table, type: 'job'|'gang'}
+RegisterNetEvent('qbx_management:client:ManageEmployee', function(data)
+    local EmployeeMenu = {}
+    local employeeLoop = data.type == 'gang' and gangs[data.work.name].grades or jobs[data.work.name].grades
+    for k, v in pairs(employeeLoop) do
+        EmployeeMenu[#EmployeeMenu + 1] = {
+            title = v.name,
+            description = 'Grade: ' .. k,
+            serverEvent = 'qbx_management:server:GradeUpdate',
+            args = {
+                cid = data.player.empSource,
+                grade = tonumber(k),
+                gradename = v.name,
+                type = data.type
+            }
+        }
+    end
+
+    EmployeeMenu[#EmployeeMenu + 1] = {
+        title = data.type == 'gang' and 'Expel Gang Member' or 'Fire Employee',
+        icon = 'fa-solid fa-user-large-slash',
+        serverEvent = 'qbx_management:server:FireEmployee',
+        args = {
+            source = data.player.empSource,
+            type = data.type
+        }
+    }
+
+    EmployeeMenu[#EmployeeMenu + 1] = {
+        title = 'Return',
+        icon = 'fa-solid fa-angle-left',
+        event = 'qbx_management:client:OpenMenu',
+        args = data.type
+    }
+
+    lib.registerContext({
+        id = 'qbx_management_open_Member',
+        title = data.type == 'gang' and 'Manage ' .. data.player.name .. ' - ' .. string.upper(QBX.PlayerData.gang.label) or 'Manage ' .. data.player.name .. ' - ' .. string.upper(QBX.PlayerData.job.label),
+        options = EmployeeMenu
+    })
+
+    lib.showContext('qbx_management_open_Member')
+end)
+
+---@param type 'job'|'gang'
+RegisterNetEvent('qbx_management:client:HireMenu', function(type)
+    local HireMenu = {}
+    local players = FindPlayers()
```

Comment:
```suggestion
    local players = findPlayers()
```

This would have to be renamed if doing my suggestion above.

Reply:
Got it. Changed.